js/redirects.js

```diff
@@ -26,6 +38,19 @@ $( document ).ready(function() {
         "to": "http://kubernetes.io/docs/whatisk8s/"
     }];
 
+    for (var i = 0; i < apiv1.length; i++) {
+        if (forwardingURL.indexOf(apiv1[i].from) > -1) {
+            var re = new RegExp(apiv1[i].pattern, 'g');
+            var matchary = re.exec(forwardingURL);
+            var newURL = apiv1[i].to;
+            if (matchary !== null) {
+                newURL += apiv1[i].postfix.replace("<token>", matchary[1]);
+            }
+            notHere = true;
+            window.location.replace(newURL);
+        }
+    }
+
```

Comment:
Agreed. This is kludgy since it's after the 404 and is done via Javascript. I have to learn more about the framework our website uses. There has to be a better way to do redirects and URL rewrites.